src/main/java/org/veupathdb/service/eda/ss/model/filter/SingleValueFilter.java

```diff
@@ -40,7 +40,7 @@ public String getSql() {
   @Override
   public Iterator<Long> streamFilteredIds(BinaryValuesStreamer binaryValuesStreamer, Study study) {
     try {
-      return binaryValuesStreamer.streamFilteredValues(this, study);
+      return binaryValuesStreamer.streamFilteredEntities(this, study);
```

Comment:
The reason I didn't like that it was called values in this particular method is because we're just outputting entity ID indexes as opposed to the values themselves.

Reply:
Maybe EntityIdIndices or something would be better